WindowsServerDocs/remote/remote-desktop-services/clients/change-listening-port.md

```diff
@@ -22,3 +23,25 @@ When you connect to a computer (either a Windows client or Windows Server) throu
 6. Close the registry editor, and restart your computer.
 
 The next time you connect to this computer by using the Remote Desktop connection, you must type the new port. If you're using a firewall, make sure to configure your firewall to permit connections to the new port number.
+
+
+You can always check the current port through PowerShell too with below commands:
+
+  ```powershell-interactive
+Get-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp' -name "PortNumber"
+Example:
+PortNumber : 3389
+PSPath : Microsoft.PowerShell.Core\Registry::HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Terminal
+Server\WinStations\RDP-Tcp
+PSParentPath : Microsoft.PowerShell.Core\Registry::HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Terminal
+Server\WinStations
+PSChildName : RDP-Tcp
+PSDrive : HKLM
+PSProvider : Microsoft.PowerShell.Core\Registry
+  ```
+You can change RDP port by executing PowerShell commands too as example shown below by adding new RDP port as 3390
+Add New RDP Port in Registry:
+  ```powershell-interactive
+Set-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp' -name "PortNumber" -Value 3390
+New-NetFirewallRule -DisplayName 'RDPPORTLatest' -Profile 'Public' -Direction Inbound -Action Allow -Protocol TCP -LocalPort 3390
+  ```
```

Comment:
```suggestion
~~~powershell
Get-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp' -name "PortNumber"
~~~
Example:
~~~powershell
PortNumber : 3389
PSPath : Microsoft.PowerShell.Core\Registry::HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Terminal
Server\WinStations\RDP-Tcp
PSParentPath : Microsoft.PowerShell.Core\Registry::HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations
PSChildName : RDP-Tcp
PSDrive : HKLM
PSProvider : Microsoft.PowerShell.Core\Registry
~~~
You can change the RDP port by executing PowerShell commands too, as in the example shown below, by adding the new RDP port as 3390.

Add New RDP Port in Registry:
~~~powershell
Set-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp' -name "PortNumber" -Value 3390
New-NetFirewallRule -DisplayName 'RDPPORTLatest' -Profile 'Public' -Direction Inbound -Action Allow -Protocol TCP -LocalPort 3390
~~~
```
The challenge for me here is to add code block fencing, because the default MarkDown code block backticks are also used to encapsulate the entire suggested change. That is why I had to use 3 tilde characters `~~~` instead of 3 backticks **```** (Tilde characters are alternate code block characters, but not necessarily the preferred ones.)

Reply:
Trond-Do you want me to do any changes now or will this be published?

Reply:
Thanks for asking. I do not have editing access in your PR. If you are able to make the changes suggested above, either by using the **commit** or **merge** feature inside my suggestion box, or by copying the changes I have outlined and applying them in your own file directly. BTW, here is the link to your PR file:
- https://github.com/vamsivellanki/windowsserverdocs/blob/vvkbranch/WindowsServerDocs/remote/remote-desktop-services/clients/change-listening-port.md

By the way, I could also create a PR directly to your file inside your own repository, if that seems more practical to you.

Reply:
Please add these suggestions by using the **[ commit ]** button feature inside my suggestion boxes. I will help you correct any glitches later on, if I have forgotten anything. For now, please just add this content by merging it manually.